src/scripts/site.js

```diff
@@ -653,7 +654,9 @@ function initClient() {
             nowPlayingHelper: componentsPath + '/playback/nowplayinghelper',
             pluginManager: componentsPath + '/pluginManager',
             packageManager: componentsPath + '/packageManager',
-            screensaverManager: componentsPath + '/screensavermanager'
+            screensaverManager: componentsPath + '/screensavermanager',
+            chromecastHelper: 'plugins/chromecastPlayer/chromecastHelpers',
```

Comment:
I moved this to the plugin so it's all self contained.

Reply:
This might be a rebasing artifact. Will fix.